### PR TITLE
Add new blog post for GSoC - First Steps

### PR DESCRIPTION
**Pull request by davidemdot**
---
title: "Google Summer of Code 2019: First Steps"
id: gsoc-2019-first-steps
---

### CODE REVIEW

File: _posts/2019-06-03-GSoC - FirstSteps.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## May 27, 2019 – Jun 02, 2019 (First week of coding)
+
+After thinking about some design and implementation details, I have been coding the back-end of the TEX parser tool.
+The tool reads each file line by line, parses and analyzes them and detects different citation formats.
+
+Code will be as self-explanatory as possible, following the contributing guidelines.
+Moreover, everything related to this project will be uploaded as a [public pull request](https://github.com/JabRef/jabref/pull/5011) so as to share code, ask and receive questions, discuss details, track the progress, etc.
+On top of that, I have been preparing some tests for these classes.
+
+Any feedback is more than welcome!
```
Comment (tobiasdiez) [inline _posts/2019-06-03-GSoC - FirstSteps.md]: Maybe move this to the end of the first paragraph (before you describe what you did)?

File: _posts/2019-06-03-GSoC - FirstSteps.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+id: gsoc-2019-first-steps
+author: "[JabRef Developers](https://github.com/JabRef/jabref/blob/master/DEVELOPERS)"
+bg: jabref-font
+color: white
+---
+
+Hello!
+My name is David Méndez, and this summer I will be working on the [LaTeX Integration project for the Google Summer of Code](https://summerofcode.withgoogle.com/projects/#6055042405105664).
+The goal of this project is to facilitate working with LaTeX for the end user.
+I will develop several tools that analyze and check LaTeX text documents: a search for bibliographic entries (which are used, how many times and where); a counting tool for words, characters and quotes per document; a validator of TEX files to solve issues; and a tool to import entries from TEX files.
```
Comment (tobiasdiez) [inline _posts/2019-06-03-GSoC - FirstSteps.md]: Make this a bullet list and maybe include one or two pictures from your prototypes.

**Comment by davidemdot** [comment]
Hope it is ready now, @tobiasdiez!

**Comment by davidemdot** [comment]
@LinusDietz, I think so. And all changes suggested by Tobias have been added.

**Comment by LinusDietz** [comment]
then let's go!

**Pull request merged**